docs/source/nodes_and_pipelines/modular_pipelines.md

```diff
@@ -1,156 +1,57 @@
-   * [Kedro-Viz](https://demo.kedro.org) accelerates development by rendering namespaced pipelines as collapsible 'super nodes'.
-
-
-## How do I create a modular pipeline?
-
-You can use a [project-specific CLI command](../development/commands_reference.md#kedro-commands) to create a modular pipeline. The pipeline name must adhere to [Python convention](https://realpython.com/python-pep8/#naming-conventions).
-
-```bash
-kedro pipeline create <pipeline_name>
+```python
+def create_new_pipeline(**kwargs) -> Pipeline:
+    return pipeline(
+    [data_science], # A name of existing pipeline
+    inputs = {"old_input__df_name" : "new_input_df_name"},
+    outputs = {"old_output_df_name" : "new_output_df_name"},
+    parameters = {"params: test_size1": "params: test_size2"},
+    )
 ```
+This means you can easily create multiple pipelines based on the `data_science` pipeline to test different approaches with various input datasets and model training parameters.
 
-```{note}
-For the full list of available CLI options, you can always run `kedro pipeline create --help` for more information.
-```
+## What is a Namespace
 
-### What does `kedro pipeline create` do?
-
-Running the `kedro pipeline create` command adds boilerplate folders and files for the designated pipeline to your project. For your convenience, Kedro gives you a pipeline-specific `nodes.py`, `pipeline.py`, parameters file and appropriate `tests` structure. It also adds the appropriate `__init__.py` files. You can see the generated folder structure below:
-
-<details>
-<summary><b>Click to see the generated folder structure</b></summary>
-
-```text
-├── conf
-│   └── base
-│       └── parameters_{{pipeline_name}}.yml  <-- Pipeline-specific parameters
-└── src
-    ├── my_project
-    │   ├── __init__.py
-    │   └── pipelines
-    │       ├── __init__.py
-    │       └── {{pipeline_name}}      <-- This folder defines the modular pipeline
-    │           ├── __init__.py        <-- So that Python treats this pipeline as a module
-    │           ├── nodes.py           <-- To declare your nodes
-    │           └── pipeline.py        <-- To structure the pipeline itself
-    └── tests
-        ├── __init__.py
-        └── pipelines
-            ├── __init__.py
-            └── {{pipeline_name}}      <-- Pipeline-specific tests
-                ├── __init__.py
-                └── test_pipeline.py
+If you need to try different options for training your model, constantly overwriting your outputs for each new pipeline can become tedious. Each model's results (outputs) should be saved in new objects, which can be annoying to manage manually. Namespaces are a perfect solution for this. By setting a `namespace="namespace_name"` parameter for each new pipeline (based on an existing pipeline), you achieve automatic output and node isolation. This is done by adding a namespace_namenamespace_name.` prefix to each pipeline node and output dataset. Your pipeline creation code will look like this:
```

Comment:
```suggestion
If you need to try different options for training your model, constantly overwriting your outputs for each new pipeline can become tedious. Each model's results (outputs) should be saved in new objects, which can be annoying to manage manually. Namespaces are a perfect solution for this. By setting a `namespace="namespace_name"` parameter for each new pipeline (based on an existing pipeline), you achieve automatic output and node isolation. This is done by adding a `namespace_name` prefix to each pipeline node and output dataset. Your pipeline creation code will look like this:
```
Might also be worth showing the example of adding the namespace prefix in the catalog and parameter files?